src/test/java/de/eposcat/master/PageTest.java

```diff
@@ -0,0 +1,206 @@
+package de.eposcat.master;
+
+
+import de.eposcat.master.approachImpl.EAV_DatabaseAdapter;
+import de.eposcat.master.approachImpl.IDatabaseAdapter;
+import de.eposcat.master.connection.H2ConnectionManager;
+import de.eposcat.master.connection.RelationalApproach;
+import de.eposcat.master.exceptions.BlException;
+import de.eposcat.master.model.Attribute;
+import de.eposcat.master.model.AttributeBuilder;
+import de.eposcat.master.model.AttributeType;
+import de.eposcat.master.model.Page;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.sql.SQLException;
+import java.util.HashMap;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+public class PageTest {
```

Comment:
The tests look good in general but it seems that multiple test cases are covered with one method.
Some test cases could be grouped together and tested in a separate method in order to increate readability and to reduce the complexity of the tests.